src/components/utils.ts

```diff
@@ -65,7 +74,7 @@ export async function handleInstall({
     })
 
     if (filePaths[0]) {
-      const path = filePaths[0]
+      const path = `${filePaths[0]}`
```

Comment:
Why this?

Reply:
Because I changed the function on the BE to install the games.
if I don't do that, if the folder have spaces, legendary will install on the wrong folder.
It comes from the folder selection like '/mnt/game folder', so that's why I need to wrap it in another string.

Reply:
So shouldn't you have done `"${...}"`?

Reply:
No, because we already get the folder name with `''`
Check this install command:
`/.../legendary install a579a08e71b341e6af106dbd170e2ab2 --base-path /mnt/ssd480/Games/Heroic/Folder with Spaces  -y |& tee "/home/flavio/.config/heroic/GamesConfig/a579a08e71b341e6af106dbd170e2ab2.log" "/tmp/heroic/install-a579a08e71b341e6af106dbd170e2ab2.sock"`

Reply:
Yeah, you were right. Fixed 😛